src/Orleans.Core/Utils/LRU.cs

```diff
@@ -157,6 +157,31 @@ public TValue Get(TKey key)
             return value;
         }
 
+        /// <summary>
+        /// Remove all expired value from the LRU instance.
+        /// </summary>
+        public void RemoveExpired()
```

Comment:
Is this only called from test code?

Reply:
No, I forgot a commit :) I just call it when we delete idle activations